### PR TITLE
Slots inheritance and properly readonly, disable and dense props passing to CountrySelect control

### LINKED ISSUE #18: There is no inheritance of slots

### PR DESCRIPTION
**Pull request by FragsterAt**
fixes #18 #19

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,1528 @@ node_modules
...
+.yarn/cache/yaml-npm-1.10.2-0e780aebdf-ce4ada136e.zip
+.yarn/cache/yargs-npm-13.3.2-1588f5dd4c-75c13e837e.zip
+.yarn/cache/yargs-npm-16.2.0-547873d425-b14afbb51e.zip
+.yarn/cache/yargs-npm-17.6.0-6343ad1a2d-604bdb4a63.zip
+.yarn/cache/yargs-parser-npm-13.1.2-546b639909-c8bb6f44d3.zip
+.yarn/cache/yargs-parser-npm-20.2.9-a1d19e598d-8bb69015f2.zip
+.yarn/cache/yargs-parser-npm-21.1.1-8fdc003314-ed2d96a616.zip
+.yarn/cache/yocto-queue-npm-0.1.0-c6c9a7db29-f77b3d8d00.zip
+.yarn/cache/yorkie-npm-2.0.0-90a2c44491-6810adaf6b.zip
+yarn.lock
```
Comment (CdTgr) [inline .gitignore:1531]:
```suggestion
```
Lock file is required

File: src/component/Index.vue [hunk 2]
```diff
@@ -30,22 +37,33 @@ export default defineComponent({
     defaultCountry: { type: String, default: () => 'us' },
     eagerValidate: { type: Boolean, default: () => true },
     useIcon: { type: Boolean, default: () => false },
+    readonly: { type: Boolean, default: () => false },
+    dense: { type: Boolean, default: () => false},
+    disable: { type: Boolean, default: () => false}
```
Comment (CdTgr) [inline src/component/Index.vue:42]:
```suggestion
    dense: { type: Boolean, default: () => false },
    disable: { type: Boolean, default: () => false },
```

File: src/component/Index.vue [hunk 3]
```diff
@@ -30,22 +37,33 @@ export default defineComponent({
...
   },
   emits: ['update:tel', 'input', 'error'],
-  setup() {
+  setup(props, { slots }) {
```
Comment (CdTgr) [inline src/component/Index.vue:45]:
```suggestion
  setup(_, { slots }) {
```

**Review by CdTgr — CHANGES_REQUESTED** [review]
Pretty good ❤️

**Review by FragsterAt — COMMENTED** [review]
Nice!

File: src/component/Index.vue (hunk 3 above)
Comment (CdTgr) [reply src/component/Index.vue:45]: @FragsterAt Missed this?

**Review by CdTgr — CHANGES_REQUESTED** [review]

**Comment by Iscrimou** [comment]
now readonly, disable and dense props are affecting only the country selector, not the input

Screenshot 2023-05-04 at 13-38-11 SocialHub Contatos
Captura de tela de 2023-05-04 13-38-23

**Pull request merged**